### ISSUE TITLE
Random internal errors after upgrading to 1.7.0

### ISSUE THREAD

**Issue by yannfill** [modoboa/modoboa#1059]
# Impacted versions

* Modoboa: 1.7.0
* installer used: No 
* Webserver: Apache

After upgrading Modoboa to 1.7.0 and following the specific instructions at http://modoboa.readthedocs.io/en/latest/upgrade.html I encounter random red popup saying "Sorry an internal error has occurred" and a blank page, or sometimes the interface is displayed without data and a "internal error" red popup.

I can sometimes display Modoboa, Statistics and Identities tabs by forcing browser refresh, but I can never display Domains tab or logs in Modoboa tab.

The Home tab display the correct number of domains and identities so I think data are not lost.

I have upgraded all the installed pip packages, except the ones with major version change:
```
bcrypt==3.1.3 is available (you have 2.0.0)
django-rest-swagger==2.1.1 is available (you have 0.3.10)
django-reversion==2.0.8 is available (you have 1.10.2)
jsonfield==2.0.0 is available (you have 1.0.3)
```
Also, I don't know if this is related, but now every pip command, Modoboa cron jobs and Apache error log spit the same warning:
```
/usr/local/lib/python2.7/dist-packages/cffi/model.py:532: UserWarning: 'point_conversion_form_t' has no values explicitly defined; guessing that it is equivalent to 'unsigned int'
  % self._get_c_name())
```

**Comment by tonioo** [comment]
I think your problems are related to this: https://github.com/modoboa/modoboa/pull/1058. Could you try to apply this patch please?

**Comment by yannfill** [comment]
Yes, everything is working again after applying the patch. Thank you.

Any clue about the cffi warning? Should I upgrade all pip packages?

**Comment by tonioo** [comment]
No, never seen this issue before.

**Comment by yannfill** [comment]
Problem solved, I had upgraded gevent to 1.2.1 but Modoboa requires ==1.1.1

I have downgraded and recompiled gevent==1.1.1 and the error is gone.
